marketing.md

```diff
@@ -0,0 +1,32 @@
+# Marketing (and sales)
+
+## Sales and marketing
+
+This page concerns sales in the sense of bringing in new projects,
+but is technically more concerned with marketing to attract clients organically.
+When members find themselves needing work, it is natural for them to shift into _sales mode_.
+However, here we talk about ongoing activity to avoid such situations.
+
+## Mindset
+
+Both the arrival of prospects and their conversion are highly stochastic processes,
+and more driven by factors outside our control than project work.
+We invest our energy in increasing the likelihood of being found/finding opportunities beyond the period of action.
+
+## Network
+
+Connections are formed at the personal level 
+and individual members should actively maintain and expand their own networks.
+Checking in on a regular basis with existing contacts is as important as making new ones.
+
+## Online presence
+
+Our website is the central portal for our brand.
+There should be uptodate content covering the following areas.
+- Who we are and how we work
+- Project examples and testimonials
+- Case studies to highlight what we can offer
+- Blog posts to demonstrate technical understanding, amongst other things
+
+Smaller content can be posted on other channels, such as LinkedIn, 
+ideally with a link back to longer content on the website.
```

Comment:
Durable high-quality content online is key to our marketing.
Areas we focus on:
- https://datavaluepeople.com
- online profiles and content directly related to members. E.g. LinkedIn, StackOverflow,...
- Blog posts or videos, posted on or off https://datavaluepeople.com/blog/